tests/test_bigbio.py

```diff
@@ -153,10 +163,21 @@ def test_metadata(self, module: ModuleType):
             metadata_attr = getattr(module, metadata_name)
 
             if metadata_name == "_LANGUAGES":
-                for lang in metadata_attr:
-                    if not isinstance(lang, metadata_type):
+
+                if not isinstance(metadata_attr, list):
```

Comment:
I have also updated the tests to make sure that  `_LANGUAGES` is not an empty list.

Reply:
:+1: